src/plugin/class-subjects-controller.php

```diff
@@ -0,0 +1,600 @@
+						'description' => __( 'Source URL from where the page was liberated', 'try_wordpress' ),
+						'type'        => 'string',
+						'context'     => array( 'view', 'edit' ),
+						'required'    => true,
+						'arg_options' => array(
+							'sanitize_callback' => 'sanitize_text_field',
+						),
+					),
+					'sourceHtml'    => array(
+						'description' => __( 'Source HTML from where the page was liberated', 'try_wordpress' ),
+						'type'        => 'string',
+						'context'     => array( 'view', 'edit' ),
+						'required'    => false,
+						'arg_options' => array(
+							'sanitize_callback' => 'sanitize_text_field',
+						),
+					),
+					'rawTitle'      => array(
+						'description' => __( 'Raw title of the page', 'try_wordpress' ),
+						'type'        => 'string',
+						'context'     => array( 'view', 'edit' ),
+						'required'    => false,
+						'arg_options' => array(
+							'sanitize_callback' => 'sanitize_text_field',
+						),
+					),
+					'parsedTitle'   => array(
+						'description' => __( 'Parsed title of the page', 'try_wordpress' ),
+						'type'        => 'string',
+						'context'     => array( 'view', 'edit' ),
+						'required'    => false,
+						'arg_options' => array(
+							'sanitize_callback' => 'sanitize_text_field',
+						),
+					),
+					'rawDate'       => array(
+						'description' => __( 'Raw date of the page', 'try_wordpress' ),
+						'type'        => 'string',
+						'context'     => array( 'view', 'edit' ),
+						'required'    => false,
+						'arg_options' => array(
+							'sanitize_callback' => 'sanitize_text_field',
+						),
+					),
+					'parsedDate'    => array(
+						'description' => __( 'Parsed date of the page', 'try_wordpress' ),
+						'type'        => 'string',
+						'context'     => array( 'view', 'edit' ),
+						'required'    => false,
+						'arg_options' => array(
+							'sanitize_callback' => 'sanitize_text_field',
+						),
+					),
+					'rawContent'    => array(
+						'description' => __( 'Raw content of the page', 'try_wordpress' ),
+						'type'        => 'string',
+						'context'     => array( 'view', 'edit' ),
+						'required'    => false,
+						'arg_options' => array(
+							'sanitize_callback' => 'sanitize_text_field',
+						),
+					),
+					'parsedContent' => array(
+						'description' => __( 'Parsed content of the page', 'try_wordpress' ),
+						'type'        => 'string',
+						'context'     => array( 'view', 'edit' ),
+						'required'    => false,
+						'arg_options' => array(
+							'sanitize_callback' => 'sanitize_text_field',
+						),
+					),
+					'transformedId' => array(
+						'description' => __( 'Post ID of transformed result of this liberated page', 'try_wordpress' ),
+						'type'        => 'string',
+						'context'     => array( 'view', 'edit' ),
+						'required'    => false,
+						'arg_options' => array(
+							'sanitize_callback' => 'sanitize_text_field',
+						),
+					),
+				),
+			),
+		);
```

Comment:
@psrpinto This is the schema with which REST API operates. Its also exposed on a url like `http://localhost:8888/wp-json/try-wp/v1/subjects/blog-post/schema`